verify_test_type/verify_test_type.sh

```diff
@@ -3,29 +3,57 @@
 # Check if all required arguments are provided
 if [ "$#" -ne 4 ]; then
   echo "Ensure that you are running the command from the repo root"
-  echo "Usage: $0 <module_name> <commit_hash> <test_name> <test_class>"
+  echo "Usage: $0 <commit_hash> <module_name> <test_class> <test_name>"
```

Comment:
Why change the order of arguments? Someone else is trying to write `README` (#1555), so you should keep in sync.

Reply:
Oh, I was not aware of the `README` change. I thought to keep the order intuitive like how we have in pr-data.csv. I have reverted it back.